core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -41,7 +41,7 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
-    $scope.myExplorationsView = 'list';
+    $scope.myExplorationsView = 'card';
```

Comment:
Ideally, this should have gone into a separate PR since it's not related to the unresolved answers work. (But you can just leave this in for this PR.)